golang/cosmos/x/vstorage/keeper/keeper.go

```diff
@@ -119,18 +119,24 @@ func NewKeeper(storeKey storetypes.StoreKey) Keeper {
 	}
 }
 
+// size_increase and size_decrease metrics represent total writes and deletes *issued*
+// respectively, which may differ from the total number of bytes committed/freed
+// to/from the store due to the store's internal implementation.
 var MetricKeyStoreSizeIncrease = []string{"store", "size_increase"}
-var MetricKeyStoreSizeDecrese = []string{"store", "size_decrease"}
+var MetricKeyStoreSizeDecrease = []string{"store", "size_decrease"}
 const MetricLabelStoreKey = "storeKey"
 
-func ReportStoreSizeMetrics(k Keeper, sizeDelta float32) {
+// reportStoreSizeMetrics exports store size increase/decrease metrics
+// when Cosmos telemetry is enabled.
+func (k Keeper) reportStoreSizeMetrics(increase int, decrease int) {
```

Comment:
Should we make those `uint`? it's not like all `int` were fitting nicely in `float32` in the first place.

```suggestion
func (k Keeper) reportStoreSizeMetrics(increase uint, decrease uint) {
```